src/oop/programs_with_multiple_classes/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,19 @@
+# oop-programs_with_multiple_classes-medium-q3
+
+## Java
+
+You want to write a program that models writing tools and stationery objects.
+It consists of a `Pencil` and a `Eraser`. See the requirements below for each class. Note that parts of each class are
+  already provided, you only need to implement the other portions.
+
+
+## Pencil
+
+Your `Pencil` class currently has the attributes `int hardness` and `boolean isForSchool`. Create a
+constructor which is accessible from another class and initializes these fields in that order.
+field.
+
+## Eraser
+
+Your `Eraser` class should have an attrirbute `double size`. Write this
```

Comment:
I feel like we should add one more thing to do for this class. Maybe add a method for them to code.

Reply:
For an idea, maybe you can add a method called `decrease` that sets the value of `size` to `size - 1`